### ISSUE TITLE
Can't Connect to Database Wordpress

### ISSUE THREAD

**Issue by NinaTanneberger** [NinaTanneberger/studio3#15]
Issue with running wordpress. Can't connect to database.

![image](https://user-images.githubusercontent.com/43361725/52409279-ca084c00-2b20-11e9-8675-f38a67aeeb10.png)

**Comment by rhaubt** [comment]
I am not sure what the issue is here. Did you follow these instructions?
https://codex.wordpress.org/Installing_WordPress

**Comment by NinaTanneberger** [comment]
Ok I fixed it. I had to create a database named "wordpress" in "MyAdmin". Then add changes to the `wp-config-sample.php` and save it as `wp-config.php`. Details of this installation will be added to the [wiki](https://github.com/NinaTanneberger/studio3/wiki/Wordpress-Installation)

![image](https://user-images.githubusercontent.com/43361725/52453915-3886f180-2b95-11e9-929a-c4d60b026033.png)

![image](https://user-images.githubusercontent.com/43361725/52453921-3fadff80-2b95-11e9-9d3a-24bef03b02c8.png)
